pkg/destroy/powervs/powervs.go

```diff
@@ -628,7 +630,7 @@ func (o *ClusterUninstaller) loadSDKServices() error {
 
 		// Get the Zone ID
 		zoneOptions := o.zonesSvc.NewListZonesOptions()
-		zoneResources, detailedResponse, err := o.zonesSvc.ListZonesWithContext(ctx, zoneOptions)
+		zoneResources, detailedResponse, err := o.zonesSvc.ListZonesWithContext(context.Background(), zoneOptions)
```

Comment:
This seems to be a different case from before. Are you sure there isn't a sane value to pass here for a timeout when listing zones?

Reply:
Good catch!